.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,26 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Version Info:**
+<!-- Provide the output from "k3s -v" -->
+K3s Version: 
+<!-- Provide the output from "uname -a" on the node(s) -->
+Node(s) OS and Version: 
```

Comment:
Could we include something like this?
**Environment:**
 - K3s version: 
 - K3s Args:
 - Node OS type and version:

Reply:
We could. Did you see I ask for k3s install args now in the steps to repro section? We had mentioned yesterday we want the steps to include everything end to end including install. Think that's okay or still prefer what you suggested?

Reply:
I would add architecture (amd64/arm/arm64)

Reply:
Updated in https://github.com/rancher/k3s/pull/1767/commits/f2cb0dc06785aace5e28dfb382e529a30d81dd32
I also added a prompt for the cluster configuration. I notice there tends to be no details on how the cluster is configured - largely how many server/agent nodes. 

I added this in - open to input on it.

Reply:
Any comments/concerns and I can unresolve the conversation.